### PR TITLE
Contracts

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: tests/Templating/Filters/AnnotationFiltersTest.php [hunk 1]
```diff
@@ -55,13 +55,13 @@ public function testAnnotationFilterWithCustom()
 	}
 
 
+	/**
+	 * @deprecated since 4.2. To be removed in 5.0.
```
Comment (GrahamCampbell) [inline tests/Templating/Filters/AnnotationFiltersTest.php]: The tests aren't part of the public api though?

Comment (GrahamCampbell) [inline tests/Templating/Filters/AnnotationFiltersTest.php]: They're in the autoload dev section.

Comment (TomasVotruba) [inline tests/Templating/Filters/AnnotationFiltersTest.php]: I don't understand, could you elaborate?

Comment (GrahamCampbell) [inline tests/Templating/Filters/AnnotationFiltersTest.php]: I'm saying what's the point in deprecating this. It's not a breaking change to remove this given that no external library is allowed to be using this class.

Comment (TomasVotruba) [inline tests/Templating/Filters/AnnotationFiltersTest.php]: I see. I think it can be removed then.

**Pull request merged**